keyboards/4by3/4by3.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_horizontal( \
```

Comment:
Not a hard requirement, but I think some block comments here regarding PCB/Pro Micro orientation in this file would help explain what's going on for anyone who may look in this file in future.

Something like this above each macro definition would be good:

```h
/* LAYOUT_horizontal
 *             ┌───┐
 *             │USB│
 * ┌───┬───┬───┼───┤
 * │K00│K01│K02│K03│
 * ├───┼───┼───┼───┤
 * │K10│K11│K12│K13│
 * ├───┼───┼───┼───┤
 * │K20│K21│K22│K23│
 * └───┴───┴───┴───┘
 */
```